### PR TITLE
Add worlddem docs

### PR DESCRIPTION
**Pull request by nekhtiari**
Pull Request
============

### Scope of the PR

Add documentation for WorldDEM 12m data block.

### Checklist

- [x] Checked spelling and grammar
- [x] Provided code samples where relevant
- [x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

**Review by rodrigoalmeida94 — APPROVED** [review]

**Review by teoEnache — APPROVED** [review]

**Review by thaisbendixen — APPROVED** [review]
LGTM!

**Review by seulgi-up42 — CHANGES_REQUESTED** [review]
Everything looks good to me, just one tiny change with "represents" to "represent" in the second sentence of the block description.

**Comment by nekhtiari** [comment]
> Everything looks good to me, just one tiny change with "represents" to "represent" in the second sentence of the block description.

fixed, thanks for catching it.

**Review by seulgi-up42 — APPROVED** [review]
Looks great! Approved!

File: source/up42-blocks/data/oneatlas-worlddem-12m.rst [hunk 1]
```diff
@@ -0,0 +1,136 @@
+.. meta::
+   :description: UP42 data blocks: WorldDEM Full Resolution
+   :keywords: UP42, data, Elevation, OneAtlas, DSM, High resolution
+
+.. _oneatlas-worlddem-12m-block:
+
+WorldDEM Full Resolution
```
Comment (markusuwe) [inline source/up42-blocks/data/oneatlas-worlddem-12m.rst:7]: Latest block naming standard says we also put the provider name here, no?

Comment (nekhtiari) [reply source/up42-blocks/data/oneatlas-worlddem-12m.rst:7]: Oh, I did not know this. I can add OneAtlas in the beginning.

**Pull request merged**